runtime/src/bank.rs

```diff
@@ -3898,6 +3921,7 @@ impl Bank {
         let collected_rent = processing_results
             .iter()
             .filter_map(|processing_result| processing_result.processed_transaction())
+            .filter_map(|processed_tx| processed_tx.executed_transaction())
```

Comment:
What happens to rent in the fee-only case? We should still be attempting to collect rent on the fee-payer, I think?

Reply:
Not really sure it matters, IIRC rent is **effectively** gone in mnb.

Reply:
I'm keeping existing behavior here. Before this change, when execution fails, we always rollback the fee payer to the pre-rent state (ie don't charge rent for transactions with a failure result). So when a transaction fails to load but can still fees, we won't be charging any rent. But yeah since rent is effectively gone, this is all moot.

Reply:
Oh. Right - we only collect rent if execution was successful. I had it in my mind that rent was charged so long as we attempted execution.

Yeah, if the existing behavior is rent only gets paid for successful execution then we should keep it that way.